README.md

```diff
@@ -381,6 +382,48 @@ As you are iterating on instrumentation you can use the `--validate-only` flag t
 ./faros_event.sh <...your command arguments...> --validate_only
 ```
 
+### Passing the request body
+
+If you would like the CLI to send an explicit request body directly to the Faros event API you can pipe the JSON event body to the script:
+
+```sh
+{
+cat <<EOF
+{
+  "type": "CD",
+  "version": "0.0.1",
+  "origin": "Faros_Script_Event",
+  "data": {
+    "deploy": {
+      "uri": "Test://test-app/Prod/1",
+      "status": "Success"
+    }
+  }
+}
+EOF
+} | ./faros_event.sh -k "<faros_api_key>"
```

Comment:
Perhaps instead just add an example of using curl like this into the README for cases which aren't supported by events api, e.g.
```
cat ... | curl -X POST https://prod.api.faros.ai/.....$FAROS_GRAPH/events -H "Authorization: $FAROS_API_KEY"
```